Mono.Cecil.Cil/Symbols.cs

```diff
@@ -824,7 +825,7 @@ static SR.AssemblyName GetSymbolAssemblyName (SymbolKind kind)
 
 			var suffix = GetSymbolNamespace (kind);
 
-			var cecil_name = typeof (SymbolProvider).Assembly ().GetName ();
+			var cecil_name = typeof (SymbolProvider).GetTypeInfo ().Assembly.GetName ();
```

Comment:
That shouldn't be necessary. `Assembly ()` here is an extension method which does exactly that.